sdk/Sdk/ExtensionsCsprojGenerator.cs

```diff
@@ -76,21 +76,20 @@ internal string GetCsProjContent()
 <Project Sdk=""Microsoft.NET.Sdk"">
     <PropertyGroup>
         <TargetFramework>{targetFramework}</TargetFramework>
-        <LangVersion>preview</LangVersion>
         <Configuration>Release</Configuration>
         <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions</AssemblyName>
-        <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions</RootNamespace>

```

Comment:
This is unrelated to the issue but wanted to take this opportunity to reduce this csproj to minimal values.